### PR TITLE
SDIT-1228 Remove unused individual calls to get booking physicalAttributes/physicalCharacteristics/physicalMarks/profileInformation

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java [hunk 1]
```diff
@@ -587,38 +580,6 @@ public List<OffenceHistoryDetail> getOffenceHistoryForBookings(@RequestBody @Par
         return bookingService.getActiveOffencesForBookings(bookingIds);
     }
 
-    @ApiResponses({
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java:590]: None of the individual endpoints were being used (but the service calls were)

File: src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java [hunk 2]
```diff
@@ -371,8 +371,6 @@ public List<Assessment> getAssessments(final Long bookingId) {
         return assessments;
     }
 
-    // @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "VIEW_PRISONER_DATA"})
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:374]: All the service calls are being called by two methods which both set the overrideRoles - so can be removed from these

File: src/test/java/uk/gov/justice/hmpps/prison/executablespecification/BookingStepDefinitions.java [hunk 3]
```diff
@@ -344,16 +344,6 @@ public void offenderIdentifiersAreRequestedForBookingId(final String bookingId)
         bookingDetail.getOffenderIdentifiers(Long.valueOf(bookingId));
     }
 
-    @When("^profile information is requested for Booking Id \"([^\"]*)\"$")
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/executablespecification/BookingStepDefinitions.java:347]: irrelevant tests removed

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.kt [hunk 4]
```diff
@@ -529,64 +533,130 @@ class BookingResourceIntTest : ResourceTest() {
...
-    val response = testRestTemplate.exchange(
-      "/api/offenders/{offenderNo}",
-      GET,
-      createHttpEntity(NORMAL_USER, null),
-      String::class.java,
-      "A1234AG",
-    )
-    assertThatJsonFileAndStatus(response, 200, "offender_extra_info.json")
-  }
+    @Test
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.kt:540]: New tests added for authorisation/authentication - ensuring that the data from 4 removed endpoints is returned in the offenderDetail for all overrideRole values

**Review by andymarke — APPROVED** [review]
lgtm

**Pull request merged**